setup.cfg

```diff
@@ -74,8 +74,8 @@ doc =
     sphinx-rtd-theme
 full =
     astropy>=4.0.1,<6.0.0
-    dask[array]>=2021.5.0
-    dask[distributed]>=2021.5.0
+    dask[array]>=2.30.0
+    dask[distributed]>=2.30.0
```

Comment:
```suggestion
    dask[array,distributed]>=2.30.0
```

Reply:
I tried that initially and the pre-commit `setup-cfg-fmt` mangled it to: `dask[array,>=2.30.0` so I just put them on separate lines. Would definitely be better on a single line though in terms of version management... any suggestion?

Reply:
I wasn't aware of this. Clearly looks like a bug in setup-cfg-fmt so, other than reporting upstream, I don't have a suggestion. Let's keep these lines separated for now since we have to.